### ISSUE TITLE
Error 500, no functionality

### ISSUE THREAD

**Issue by Reoga** [filebrowser/filebrowser#3195]
**Description**
Trying to create a file share on my pi4. Followed https://pimylifeup.com/raspberry-pi-file-browser/ tutorial. Removed the version line from compose. Updated everything. Page loads. Login works. Second drive missing on web page. Can't upload to server, gets no error, just nothing happened. Cant make folders, or files. No errors, just nothing happens. Tried to add user, get error 500. Checked permissions, then elevated admin to root access. No change, still get error 500, and cant use the file system.

**Expected behaviour**
System to let me upload, and make users.

**What is happening instead?**
No error for upload, or adding folders, but also nothing happens. Error 500 on user creation.

**Additional context**
New install. Followed tutorial. Tried reinstall, and tried multiple other drives. Tried alternative tutorials. Same result. Also tried a file manager that had FileBrowser as a built in plugin, had same issue.

**How to reproduce?**
Follow tutorial.

**Files**
![Screenshot 2024-05-05 032313](https://github.com/filebrowser/filebrowser/assets/38743578/ea718f34-88d4-4328-8bb0-aea23fb83951)
![Screenshot 2024-05-05 032408](https://github.com/filebrowser/filebrowser/assets/38743578/2e138417-9eb5-4055-b7bf-70041b98d318)
![Screenshot 2024-05-05 032529](https://github.com/filebrowser/filebrowser/assets/38743578/a71af33b-2839-40fc-a0c2-765ed5280923)

**Comment by Reoga** [comment]
Oh yeah, using v2.29.0 of FileBrowser

**Comment by kloon15** [comment]
Did u create the database.db file beforehand? Docker has a habit or creating directory mount if no such file exists.
Also u didnt mount ur 2 shares under /srv which is most likely ur issue.

**Comment by Reoga** [comment]
> Did u create the database.db file beforehand? Docker has a habit or creating directory mount if no such file exists. Also u didnt mount ur 2 shares under /srv which is most likely ur issue.

Thank you for the response.
I am trying to figure out what you are talking about.
So, I verified the database.db exists. It is in the FileBrowser dir; but just in case, I made another using the same code at the root as the starting point. The tutorial is unclear on the place to run the script from. I just assumed it was from the FileBrowser dir as that is were you are when you get to the step.

I have never opened the database.db file, so i don't know what is in it.

I am 100% sure it is mounted, but, if you think that will help, I unmounted the drive, and created a new mount.

No change to FileBrowser, still no function, or second drive.

some screenshots to show the changes. At this point, I think I would benefit from some examples if I did something wrong. 

As far as the drives: no mater how I mount them, nothing changes.
As far as the database.db: It seems to be none functional. I even tried installing mysql to see if that would solve it. I know it is not in the instructions to do so, but I am pulling my hair out at this point.

![1](https://github.com/filebrowser/filebrowser/assets/38743578/201c7500-2c09-4f66-b535-69577cecc14f)
![2](https://github.com/filebrowser/filebrowser/assets/38743578/9915fc81-cffb-4d50-af9a-99706c586c05)
![3](https://github.com/filebrowser/filebrowser/assets/38743578/e0b5baf0-7422-4eb4-9946-43eb10e4f663)
![4](https://github.com/filebrowser/filebrowser/assets/38743578/2f167b81-0b95-48b2-8127-fdd62f26fd76)

**Comment by kloon15** [comment]
U didnt specify a target for the first 2 mounts, heres an example with 2 different locations mounted under /srv:
```
services:
  filebrowser:
    volumes:
      - /home/nginx/filebrowser:/srv
      - /home/qbittorrent/Downloads:/srv/Torrents
```

**Comment by Reoga** [comment]
You are absolutely correct. (thank you for the example, that helped a lot!)
I have good news!
Last night around 2am I was getting ready to try and follow another tutorial for an alternate file share, and started with update, as you do. Well, I do it first, then again after right before deployment. 

Anyways: It pulled a 33kb update from FileManager master repository. So I tried to deploy it again, just to see if something changed, and got an error for drive mounting. My second drive had been unmounted. (never got any errors for mounting prior to this, even when it was absurdly wrong). Saw your post, and corrected the mount pointer. (Thank you)
 So I mounted it again, and got another error for a file permission/access denied. Remember, I elevated main user to root. (i know, bad idea) The error said it couldn't write to user profile for permissions. So, i changed the account back to a sudo user. Rebooted.

These are all new errors I was not getting before, so I am getting exited at this point. I can work with errors. No errors are a pain.

Well, I ran the server, and tried to upload a photo, and it worked. Tried to make a folder, and success! Made new users.... and no 500 error!!!
AND: My second drive is there!!, however the main drive with the OS is no longer showing up, but who cares!! It is just a 128gb main drive. The second one is the one i wanted to use as the repository for my files, 8TB. Works great now. I have no idea what was done in that 33kb update, but i suspect it was someone on the repository side that saw something wrong, then corrected it. That person is my hero!
